pkg/environments/gitops.go

```
@@ -233,6 +237,30 @@ func ModifyChartFiles(dir string, details *PullRequestDetails, modifyFn ModifyCh
 	return nil
 }
 
+func mergeNestedAppsValues(dir string, dependencies []*helm.Dependency, values map[string]interface{}) {
+	for _, dep := range dependencies {
+		if dep.Name != "jenkins-x-platform" {
+			valuesFilePath := filepath.Join(dir, dep.Name)
+			valuesFile, err := helm.FindValuesFileName(valuesFilePath)
+			if err != nil {
+				valuesFilePath = filepath.Join(dir, helm.DefaultEnvironmentChartDir, dep.Name)
+				valuesFile, err = helm.FindValuesFileName(valuesFilePath)
+				if err != nil {
+					log.Warnf("Unable to find the values.yaml file in path: %s", valuesFilePath)
```

Normally you want to return the error rather than log it